### PR TITLE
initial build out of table components, updates to webpack and dependencies.

### PR DESCRIPTION
**Pull request by TravisSalad**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,31 +1,34 @@
...
+// import createStore from './Services/Store';
+// import createHistory from 'history/createBrowserHistory';
+// import { syncHistoryWithStore } from 'react-router-redux';
+// import { Provider } from 'react-redux';
+// import { BrowserRouter as Router } from 'react-router-dom';
+// import publicRoutes from './routes';
+// import axe from 'react-axe';
+// import App from './App';
+import Table from './Components/Table';
+export default { Table };
```
Comment (canfie1d) [inline src/index.js]: I thought `default` broke this?

File: stories/index.js [hunk 2]
```diff
@@ -4,6 +4,8 @@ import { storiesOf } from '@storybook/react';
 import { action } from '@storybook/addon-actions';
 import { linkTo } from '@storybook/addon-links';
 import Landing from '../src/Components/Landing';
+import Table from '../src/Components/Table';
+import { tableColumns, tableData } from '../src/Services/Constants';
```
Comment (canfie1d) [inline stories/index.js]: Do you think that each component should have its own data file?

**Review by canfie1d — COMMENTED** [review]
Just a couple Qs

File: src/index.js (hunk 1 above)
Comment (TravisSalad) [reply src/index.js]: you're right.. I have more I forgot to commit.  Adding it now.

File: stories/index.js (hunk 2 above)
Comment (TravisSalad) [reply stories/index.js]: yeah good call. Will update.

**Review by canfie1d — APPROVED** [review]
👍

**Pull request merged**